arch/xtensa/src/common/xtensa_tcbinfo.c

```diff
@@ -63,7 +63,8 @@ const struct tcbinfo_s g_tcbinfo =
   .state_off = TCB_STATE_OFF,
   .pri_off   = TCB_PRI_OFF,
   .name_off  = TCB_NAME_OFF,
-  .reg_num   = XCPTCONTEXT_REGS,
+  .basic_num = XCPTCONTEXT_REGS,
+  .total_num = XCPTCONTEXT_REGS,
```

Comment:
Can you please also add the `regs_off` member?

Reply:
@xiaoxiang781216 @zhuyanlinzyl can we please get this one done and hopefully merge this PR?

Reply:
Done.